src/mgmt/cli_mgmt.c

```diff
@@ -653,6 +655,18 @@ dc_mgmt_net_cfg_init(const char *name, crt_init_options_t *crt_info)
 			D_GOTO(cleanup, rc = d_errno2der(errno));
 	}
 
+	if (info->client_firewall_mode) {
+		rc = asprintf(&client_firewall_mode, "%d", info->client_firewall_mode);
```

Comment:
More a question for @mjmac 

Is there a reason we don't use D_ASPRINTF/D_FREE?

Reply:
Maybe it has something to do with how `d_freeenv_str()` works? @knard38 reworked all of that stuff last year for thread safety reasons, iirc.